winter/exceptions/throws.py

```diff
@@ -0,0 +1,38 @@
+import typing
+from collections import defaultdict
+from typing import DefaultDict
+from typing import Set
+from typing import Type
+
+import dataclasses
+
+from ..core import ComponentMethod
+from ..core import annotate
+
+if typing.TYPE_CHECKING:
+    from .handlers import ExceptionHandler
+
+_throws: DefaultDict[object, Set[Type[Exception]]] = defaultdict(set)
```

Comment:
this need delete

Reply:
not sure I understand

Reply:
if you're saying it should be deleted, then I just did it :)